pmd-java/src/main/java/net/sourceforge/pmd/lang/java/oom/rule/CyclomaticComplexityRule.java

```diff
@@ -28,21 +28,21 @@
  */
 public class CyclomaticComplexityRule extends AbstractJavaMetricsRule {
 
-    public static final IntegerProperty REPORT_LEVEL_DESCRIPTOR = new IntegerProperty(
+    private static final IntegerProperty REPORT_LEVEL_DESCRIPTOR = new IntegerProperty(
```

Comment:
Hmm well maybe we should make them protected, but I think there's no point in having them public. The class is not designed for inheritance though, so maybe we should make it final? I haven't seen many `final` rules in PMD...